src/containers/dashboard/SearchCases.js

```diff
@@ -0,0 +1,150 @@
+// @flow
+import React, { useEffect, useState } from 'react';
+
+import styled from 'styled-components';
+import { List, Map } from 'immutable';
+import {
+  Button,
+  CheckboxSelect,
+  Input,
+  Label,
+  PaginationToolbar,
+  SearchResults,
+} from 'lattice-ui-kit';
+import { LangUtils, ReduxUtils } from 'lattice-utils';
+
+import { SEARCH_CASES, searchCases } from './actions';
+import { CASES_RESULT_LABELS } from './constants';
+
+import { DashboardReduxConstants, ProfileReduxConstants, REQUEST_STATE } from '../../core/redux/constants';
+import { goToRoute } from '../../core/router/RoutingActions';
+import { getPersonName } from '../../utils/people';
+import { MAX_HITS } from '../../utils/search/constants';
+import { useDispatch, useSelector } from '../app/AppProvider';
+import { getStaff } from '../profile/src/actions';
+import { EMPTY_VALUE, RoleConstants } from '../profile/src/constants';
+
+const { DASHBOARD, SEARCHED_CASES_DATA, TOTAL_CASES_HITS } = DashboardReduxConstants;
+const { PROFILE, STAFF_MEMBERS } = ProfileReduxConstants;
+const { CASE_MANAGER } = RoleConstants;
+const { isFailure, isPending, isSuccess } = ReduxUtils;
+const { isDefined } = LangUtils;
+
+const NO_CASE_MANAGER = 'No Case Manager';
+
+const SearchGrid = styled.div`
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: 2fr 1fr;
+  width: 100%;
```

Comment:
No change required, but since lattice-ui now imports all of material-ui, this could probably be replaced with a `Grid` component.